packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1722,9 +1722,12 @@ mixin WidgetInspectorService {
     return _safeJsonEncode(_getProperties(diagnosticsNodeId, groupName));
   }
 
-  List<Object> _getProperties(String? diagnosticsNodeId, String groupName) {
-    final DiagnosticsNode? node = toObject(diagnosticsNodeId) as DiagnosticsNode?;
-    return _nodesToJson(node == null ? const <DiagnosticsNode>[] : node.getProperties(), InspectorSerializationDelegate(groupName: groupName, service: this), parent: node);
+  List<Object>  _getProperties(String? diagnosticsOrDiagnosticableId, String groupName) {
+    final DiagnosticsNode? node = _idToDiagnosticsNode(diagnosticsOrDiagnosticableId);
+    if (node == null) {
+      return const <DiagnosticsNode>[];
```

Comment:
this type is wrong. Should be List<Map<String, Object?> to match _nodesToJson.

Reply:
Should I change contract of the method that returns List?
Changed DiagnosticsNode to Object